File: Tasks/MySolution/random.java
package Tasks.MySolution;

public class random {
    public static void main(String[] args) {
        int result = randomProcess(1, 9);
        System.out.println(result);
    }

    public static int randomProcess(int min, int max) {
        int preResult = (int) (Math.floor(Math.random() * (max - min + 1)));
        return min + preResult;
    }

    public static int randomProcess(int max) {
        int preResult = (int)  (Math.floor(Math.random() * (max + 1)));
        return 0;
    }
}

File: NotFormatted/random.js
// Generate random integer value in given range

Random = (min, max) => {
if ( max === undefined ){;
    max = min; return Math.floor(Math.random() * (max + 1));
    ;} else {;
    return min + Math.floor(Math.random() * (max - min + 1));
};
};

require('../Tests/random.js')(Random);
